src/components/IssueUI/IssueRequestStatusUI/ReceivedIssueRequest/index.tsx

```diff
@@ -1,19 +1,18 @@
 import { useTranslation } from 'react-i18next';
 import clsx from 'clsx';
-import { useSelector } from 'react-redux';
-import { useQuery } from 'react-query';
 import { useErrorHandler, withErrorBoundary } from 'react-error-boundary';
 
 import RequestWrapper from 'pages/Bridge/RequestWrapper';
 import ExternalLink from 'components/ExternalLink';
 import ErrorFallback from 'components/ErrorFallback';
 import Ring48, { Ring48Title, Ring48Value } from 'components/Ring48';
 import { BTC_EXPLORER_TRANSACTION_API } from 'config/blockstream-explorer-links';
-import { POLKADOT, KUSAMA } from 'utils/constants/relay-chain-names';
 import { shortAddress } from 'common/utils/utils';
-import genericFetcher, { GENERIC_FETCHER } from 'services/fetchers/generic-fetcher';
-import { StoreType } from 'common/types/util.types';
 import { getColorShade } from 'utils/helpers/colors';
+import { POLKADOT, KUSAMA } from 'utils/constants/relay-chain-names';
+import useCurrentActiveBlockNumber from 'services/hooks/use-current-active-block-number';
+import useStableBitcoinConfirmations from 'services/hooks/use-stable-bitcoin-confirmations';
+import useStableParachainConfirmations from 'services/hooks/use-stable-parachain-confirmations';
```

Comment:
Replaced with these reusable hooks I had created in the process of working on the tx page.